ProofPoint/proofpoint-pod/_meta/fields.yml

```diff
@@ -93,6 +93,24 @@ proofpoint.pod.quarantine.folder:
   description: "The quarantine folder containing the copy of the message"
   type: keyword
 
+proofpoint.pod.smtp.sender:
```

Comment:
couldn't this field be a `email.sender.address`  and `recipient` be `email.from.address`

Reply:
Good question that I also asked myself, but,
according to the [ECS specification](https://www.elastic.co/guide/en/ecs/8.4/ecs-email.html) and the [RFC 5322](https://www.rfc-editor.org/rfc/rfc5322), `email.sender.address` should host the email header `Sender` and `email.sender.address` the email header `From`.
The extracted information are from the SMTP level and come from the command `MAIL FROM` for the sender and `RCPT TO` for the recipients.